src/Writers/LaTeXWriter.jl

```diff
@@ -180,6 +188,19 @@ function compile_tex(doc::Documents.Document, settings::LaTeX, texfile::String)
                    "Logs and partial output can be found in $(Utilities.locrepr(logs))." exception = err
             return false
         end
+    elseif settings.platform == "tectonic"
+        @info "LaTeXWriter: using tectonic to compile tex."
+        tectonic = isnothing(settings.tectonic) ? Sys.which("tectonic") : settings.tectonic
+        isnothing(tectonic) && (@error "LaTeXWriter: tectonic command not found."; return false)
+        try
+            piperun(`$(tectonic) -X compile --keep-logs -Z shell-escape $texfile`)
+            return true
+        catch err
+            logs = cp(pwd(), mktempdir(); force=true)
```

Comment:
```suggestion
            logs = cp(pwd(), mktempdir(); force=true)
```
Isn't that dir deleted when Julia exits? Can you also print the output for people who want to debug CI logs? Maybe also use the `--print` flag. That one has helped me a lot in some situations when debugging Books.jl.

Reply:
Yes, according to the docs, the temporary directory is deleted when Julia exits. Should we fix it also for the "native" (latexmk) case?

What output are you referring to? That of the `tectonic` command? If I understand `piperun` correctly, the output is visible if you pass `--verbose` to `ARGS`. Otherwise, it gets redirected to files.

Reply:
Yes, I meant for `tectonic`.

Reply:
> Should we fix it also for the "native" (latexmk) case?

I don't know. I only know things about Tectonic and not much about Documenter

Reply:
#1697